allure-framework-modules/allure-webapi-module/src/main/java/com/capgemini/ntc/example/core/base/properties/PropertiesFileSettings.java

```diff
@@ -0,0 +1,31 @@
+package com.capgemini.ntc.example.core.base.properties;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+
+public class PropertiesFileSettings {
```

Comment:
Overall question,  what parameters you would need to have in WebAPI startup. 
These parameters are taken from "settings.properties" file 

https://github.com/devonfw/devonfw-testing/blob/feature/webApiModule/allure-framework-modules/allure-webapi-module/src/resources/settings.properties

Reply:
Under consideration in future